package pkgExceptionHandling;
import java.util.Random;

public class RandomDivisionEx {
	/**
	 * This method generates two random numbers and performs division on them.
	 * In case of exception, it is handled by try-catch block.
	 */
	public int randomDiv() {
		Random rndNum = new Random();  // creating object of Random class to generate random numbers
		int num1 = rndNum.nextInt(1);
		int num1 = rndNum.nextInt(10);
	}
}
